include/sys/sched.h

```diff
@@ -105,6 +105,12 @@ long sched_switch(void);
  */
 void sched_maybe_preempt(void);
 
+/*! \brief Unconditionally yield the CPU to another thread.
+ *
+ * \note Panics if interrupts or preemption are disabled.
+ */
+void sched_yield(void);
```

Comment:
`sched_yield` is defined in `include/sched.h` both on _Linux_ and _BSD_ systems.

Reply:
I'm in favour of hiding most declarations from this file from user-land programs with simple trick we use in other header files: http://bxr.su/NetBSD/sys/sys/sched.h#143 (`#ifdef _KERNEL`)

Reply:
...my bad, I forgot to add the `include/sched.h` file to the commit.